akka-docs/src/main/paradox/stream/operators/Source/completionStageSource.md

```diff
@@ -6,6 +6,21 @@ Streams the elements of an asynchronous source once its given *completion* opera
 
 ## Signature
 
+@@@ div { .group-scala }
+
+@@snip[Source.scala](/akka-stream/src/main/scala/akka/stream/javadsl/Source.scala) { #completionStageSource }
```

Comment:
This sounds like one of those methods where there *is* a scaladsl variant, but it's named differently (Source.future). Is that indeed the equivalent, and if so, could we merge the 2 documentation pages? (I think there's some examples of this, perhaps toSeq?)

Reply:
Ah, the `future.md` page has a mention to `completionStage`.

Hold on, I'm interested in `futureSource.md` though. Hmm, OK, I think I'll clean all those 4 pages at once...